app/presenters/error_presenter.rb

```diff
@@ -1,18 +1,26 @@
 class ErrorPresenter
   attr_reader :error_details
 
-  def initialize(claim, message_file = nil)
-    @claim = claim
-    @errors = claim.errors
-    message_file ||= "#{Rails.root}/config/locales/error_messages.#{I18n.locale}.yml"
+  def initialize(object, message_file = nil)
+    @object = object
```

Comment:
I don't think that `@object` is actually used anywhere so this line can be removed.